app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
```

Comment:
must be blank lines between different import types.
import also must be sorted alphabetically